assembl/static2/tests/unit/components/__snapshots__/search.spec.jsx.snap

```diff
@@ -65,26 +65,6 @@ exports[`Hits components DumbExtractHit component should render an extract hit 1
         </div>
       </div>
     </div>
-    <div
-      className="link-idea"
-    >
-      <div
-        className="label"
-      >
-        <span>
-          This post is related to the following themes:
-        </span>
-      </div>
-      <div
-        className="badges"
-      >
-        <span
-          className="badge"
-        >
-          Interesting thematic
-        </span>
-      </div>
-    </div>
```

Comment:
I know the snaphost is wrong here because of the way we import Link. Nothing we can do to fix it.

Reply:
Oooh that's why it removed those lines. I first thought it was normal and then by reviewing my code I figured I had wrongly updated the snapshots and was wondering what happend here but now it makes sense